packages/nimble-components/src/theme-provider/design-tokens.ts

```diff
@@ -218,6 +214,67 @@ export const drawerWidth = DesignToken.create<string>(
     styleNameFromTokenName(tokenNames.drawerWidth)
 ).withDefault('784px');
 
+// Drop Shadow Tokens
+export const [
+    elevation1BoxShadow,
+    elevation1BoxShadowColor,
+    elevation1BoxShadowXOffset,
+    elevation1BoxShadowYOffset,
+    elevation1BoxShadowBlurSize
+] = createDropShadowTokens(
+    tokenNames.elevation1BoxShadow,
+    (element: HTMLElement) => hexToRgbaCssColor(getColorForTheme(element, Black, Black, Black), 0.16),
+    '0px',
+    '1px',
+    '4px'
+);
+
+export const [
+    elevation2BoxShadow,
+    elevation2BoxShadowColor,
+    elevation2BoxShadowXOffset,
+    elevation2BoxShadowYOffset,
+    elevation2BoxShadowBlurSize
+] = createDropShadowTokens(
+    tokenNames.elevation2BoxShadow,
+    (element: HTMLElement) => hexToRgbaCssColor(getColorForTheme(element, Black, Black, Black), 0.16),
+    '0px',
+    '2px',
+    '4px'
+);
+
+export const [
+    elevation3BoxShadow,
+    elevation3BoxShadowColor,
+    elevation3BoxShadowXOffset,
+    elevation3BoxShadowYOffset,
+    elevation3BoxShadowBlurSize
+] = createDropShadowTokens(
+    tokenNames.elevation3BoxShadow,
+    (element: HTMLElement) => hexToRgbaCssColor(getColorForTheme(element, Black, Black, Black), 0.2),
+    '0px',
+    '3px',
+    '8px'
+);
+
+export const [
+    elevation4BoxShadow,
+    elevation4BoxShadowColor,
+    elevation4BoxShadowXOffset,
+    elevation4BoxShadowYOffset,
+    elevation4BoxShadowBlurSize
+] = createDropShadowTokens(
+    tokenNames.elevation4BoxShadow,
+    (element: HTMLElement) => hexToRgbaCssColor(getColorForTheme(element, Black, Black, Black), 0.3),
+    '0px',
+    '4px',
+    '8px'
+);
+
+export const elevation3FullHeightBoxShadowXOffset = DesignToken.create<string>(
```

Comment:
Sorry to keep shifting around, naming is hard 😅

Now I'm wondering if we really need to break-out all the parts of the drop shadow, are they needed outside nimble or do we know of upcoming usages?

Maybe we can just have the tokens (without all the subparts exposed):

```
elevation1BoxShadow
elevation2BoxShadow
elevation3BoxShadow
elevation4BoxShadow
elevation3FullHeightLeftBoxShadow
elevation3FullHeightRightBoxShadow
```

Reply:
I think you're right. We don't have a good use case for needing all the subparts exposed as long as we handle a few special cases for slide outs.

I did a search for shadows in SLE, and I realized that I had the wrong values for elevation 3 (in this branch, the value is `#00000033 0px 3px 8px`). It is used for the drawer, so it shouldn't have a Y offset. But, for the navigation panel in SLE, it should have a Y offset in order to keep the shadow from overlapping with the header bar.

According to the visual design specs, the correct values are:
- `#00000033 -3px 0px 8px` for right drawer
- `#00000033 3px 0px 8px` for left drawer
- `#00000033 3px 4px 8px` for navigation pane

Some options I can see for resolving this problem are:
_Option 1_
- Remove the elevation 3 box shadow (and rename elevation 4 to elevation 3)
- Hard-code the box shadow in the drawer & side nav with the justification that these are component-specific and don't need to be generic tokens
    - We can get away with this because the color isn't theme-specific (it's always #00000033). If it were theme-specific, it would, that would be difficult to hard-code in SLE
- Don't export tokens for the subparts of the box shadows

_Option 2_
- Add more variants for elevation 3, such as `elevation3FullHeightLeftBoxShadow`, `elevation3FullHeightRightBoxShadow`, `elevation3AlmostFullHeightLeftBoxShadow` -- the last one obviously would need a better name
- Don't export tokens for the subparts of the box shadows

_Option 3_
- Keep the all the tokens for subparts of the box shadow
- Export any necessary tokens for parts that change (i.e. all the different values for the full-height/navigation/left/right)

I like option 1, but I'm curious for your opinion. With option 1, if hard-coding the values within nimble & SLE ever become insufficient, then we could revisit and potentially add addition tokens named something like `drawerRightBoxShadow`, `drawerLeftBoxShadow`, and `navigationLeftBoxShadow` at that time.

Reply:
Yea I think Option 1 makes sense with the future addendum of adding tokens for some very specific contexts if they end up being needed externally in more places. I'm sold 👍